app/src/modules/vega-chart.js

```diff
@@ -0,0 +1,116 @@
+import { literal, namedNode } from '@rdfjs/data-model'
+import { fromRdf } from 'rdf-literal'
+import { querySparql } from '@/modules/sparql'
+
+const defaultQuery = `
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+SELECT DISTINCT ?c (COUNT(?x) AS ?count) (MIN(?class) AS ?class) 
+WHERE {
+    ?x a ?c.
+    ?c rdfs:label ?class.
+}
+GROUP BY ?c
+ORDER BY DESC(?count)
+LIMIT 10
+`.trim()
+
+const defaultSpec = {
+  $schema: 'https://vega.github.io/schema/vega-lite/v4.json',
```

Comment:
we should probably change the default spec to use vega lite v5 now that we support it.

Reply:
made this change in [b2d6c1d](https://github.com/Duke-MatSci/materialsmine/pull/106/commits/b2d6c1d67191b5970f9eb19a6693ce2d7c46b79a)